packages/react/src/themes/teams/components/Carousel/carouselStyles.ts

```diff
@@ -10,18 +10,22 @@ const carouselStyles: ComponentSlotStylesPrepared<
   root: (): ICSSInJSStyle => ({
     display: 'inline-block',
   }),
-  itemsContainerWrapper: ({ variables: v }): ICSSInJSStyle => ({
+  itemsContainerWrapper: ({ variables: v, props: p }): ICSSInJSStyle => ({
     display: 'flex',
     width: pxToRem(v.width),
     overflowX: 'hidden',
+    border: '1px solid transparent',
+
+    ...(p.shouldFocusContainer && {
```

Comment:
Should we use there `focus-visible`?

Reply:
This is a valid point, we should not have this border when it is coming from click, but unfortunately we have to use `isFromKeyboard` :(